tools/envoy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM allegro/envoy-dev:v1.16.1-dev-lua-segfault-fix-1-16-0-backport-20201118-df9dc819
+FROM envoyproxy/envoy-alpine:v1.17.1
```

Comment:
Let's keep in sync with https://github.com/allegro/envoy-control/blob/master/envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt#L37

Reply:
updating here would require bigger investment, after updating and customization in 
package manager https://github.com/allegro/envoy-control/blob/master/envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt#L22  to RUN apk add --no-cache curl iproute2 iptables dnsmasq i couldn't make it working as we have custom setup